packages/server/utils/getJiraCloudIdAndKey.ts

```diff
@@ -1,2 +1,5 @@
-const getJiraCloudIdAndKey = (id: string) => id.split(':')
+const getJiraCloudIdAndKey = (id: string): [string, string] => {
+  const [cloudId, key] = id.split(':')
+  return [cloudId, key]
```

Comment:
`return [cloudId, key] as [cloudId: string, issueKey: string]` takes it a step further. then, the person calling this will get a helpful little popup tells them which is which

Reply:
ooh, nice touch. Updated!

Reply:
Hmm it looks like this update causes a Typecheck error. TypeScript isn't showing any errors in the file, but I'm getting a build failure. Looking at [the docs](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements), it looks like the syntax is correct and we're using TypeScript 4.0. 
Perhaps there's a problem with how the build is set-up? I'll keep digging

Reply:
can you bump all typescript versions to latest in the package.jsons? (can't trust the `^`). running `yarn outdated` shows that some are referencing v3